pages/AddSurveyQuestionCommonUtils.js

```diff
@@ -0,0 +1,58 @@
+const { expect } = require("@playwright/test");
+
+export async function reset(page){
+    await page.getByRole('button', { name: 'Reset', exact: true }).click();
+    await expect(page.getByRole('heading', { name: 'Reset Confirmation' })).toBeVisible();
+    await expect(page.getByLabel('Reset Confirmation')).toContainText('Are you sure you want to reset? Once the reset is done, the data cannot be retrieved back.');
+    await expect(page.getByRole('button', { name: 'Cancel' })).toBeEnabled();
+    await expect(page.getByRole('button', { name: 'Reset' })).toBeEnabled();
+    await page.getByRole('button', { name: 'Reset' }).click();
+  }
+
+export async function questionFieldsCharacterLimit(page){
```

Comment:
```suggestion
export async function checkQuestionFieldsCharacterLimit(page){
```

The function name in this case is an action. Mention what you are doing in that function